components/tag-list/tag-list-item-mixin.js

```diff
@@ -65,12 +63,27 @@ export const TagListItemMixin = superclass => class extends superclass {
 		super();
 		/** @ignore */
 		this.role = 'listitem';
+		this._id = getUniqueId();
+	}
+
+	firstUpdated(changedProperties) {
+		super.firstUpdated(changedProperties);
+
+		const container = this.shadowRoot.querySelector('.tag-list-item-container');
+		this.addEventListener('focus', () => container.focus());
+		this.addEventListener('blur', () => container.blur());
 	}
 
-	_renderTag(tagContent) {
+	_renderTag(tagContent, hasTruncationTooltip) {
+		const tooltip = hasTruncationTooltip ? html`
+				<d2l-tooltip for="${this._id}" show-truncated-only>
+					${tagContent}
+				</d2l-tooltip>
+			` : null;
 		return html`
-			<div class="tag-list-item-container d2l-label-text">
-				<div class="tag-list-item-content">${tagContent}</div>
+			${tooltip}
+			<div class="tag-list-item-container d2l-label-text" id="${this._id}" tabindex="0">
```

Comment:
Do we need the container itself to be tabbable? Previously we had the first tag have a tabindex of 0 and the rest with -1.

Reply:
Could we use tabindex="-1" instead? That means it is focusable via script (so our arrowkeys mixin), but not via tab.

Reply:
Yup that works!